.github/workflows/health-check.yaml

```diff
@@ -56,16 +56,38 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Get changed files
+        id: changed-files
+        uses: step-security/changed-files@v45
+        with:
+          files: |
+            *.env
+            *.repos
+            .github/actions/docker-build/action.yaml
+            .github/workflows/health-check.yaml
+            ansible-galaxy-requirements.yaml
+            ansible/**
+            docker/**
```

Comment:
Thanks for this improvement, it will save us a lot of time!

Maybe we can add:
- https://github.com/autowarefoundation/autoware/blob/main/setup-dev-env.sh This is being utilized in many places in the main Dockerfile.
- https://github.com/autowarefoundation/autoware/blob/main/ansible-galaxy-requirements.yaml if this was broken, the ansible scripts may have issues?

Reply:
The `setup-dev-env.sh` was appended. https://github.com/autowarefoundation/autoware/pull/6055/commits/5bea8e90c34947ca28df268e8aa87ff3b836c264

> https://github.com/autowarefoundation/autoware/blob/main/ansible-galaxy-requirements.yaml if this was broken, the ansible scripts may have issues?

I think so but I need to learn Ansible more.